packages/govuk-frontend/src/govuk/components/accordion/accordion.mjs

```diff
@@ -486,6 +486,10 @@ export class Accordion extends GOVUKFrontendComponent {
    * @param {boolean} expanded - Section expanded
    */
   updateShowAllButton(expanded) {
+    if (!this.$showAllButton || !this.$showAllText || !this.$showAllIcon) {
+      return
+    }
```

Comment:
**question** We're creating these elements ourselves, what's requiring us to add this early return?

Reply:
Ah we [discussed via Slack thread](https://gds.slack.com/archives/C05P6FJ1YH2/p1696235949838579?thread_ts=1695976340.212499&cid=C05P6FJ1YH2) to use silent returns (not errors) for self-created elements

I've been running [ESLint `@typescript-eslint/stylistic-type-checked`](https://typescript-eslint.io/linting/configs/#projects-with-type-checking) with @domoscargin in https://github.com/alphagov/govuk-frontend/pull/4106 so we can teach ESLint to be aware of types. These three are all `null` at instantiation

Reply:
Returning rather than throwing is fine as we're creating these elements, as discussed on the Slack thread. 

Shame TypeScript needs an extra nudge leading to us checking things we do know exist 😔

Reply:
Yeah it's a shame, but good to be consistent

Alternatively we could consider getters and avoid `null` defaults entirely?

Here's a diff of what that would look like:
https://github.com/alphagov/govuk-frontend/compare/null-checks...null-checks-getters